e2e/testcases/oci_sync_test.go

```diff
@@ -442,9 +378,9 @@ func getImageDigest(nt *nomostest.NT, imageName string) (string, error) {
 	return hex, nil
 }
 
-// imageDigestFunc wraps getImageDigest to return a Sha1Func that caches the
+// fetchImageDigestFunc wraps getImageDigest to return a Sha1Func that caches the
 // image digest, to avoid polling the image registry unnecessarily.
-func imageDigestFunc(imageName string) nomostest.Sha1Func {
+func fetchImageDigestFunc(imageName string) nomostest.Sha1Func {
```

Comment:
Probably woulda called these funcs `imageDigestFuncByName` and `imageDigestFuncByDigest`. 

It took me a while to figure out why you had renamed cause i found `fetchImageDigestFunc` confusing... It's not fetching the func, it's getting a func to fetch (and cache) the digest.

Reply:
done